### DISCUSSION TITLE
Alt key for Macos users in ghostty terminal

### DISCUSSION THREAD

**Discussion by merikan** [zellij-org/zellij#3895] [General]
You can use the [`macos-option-as-alt` option](https://ghostty.org/docs/config/reference#macos-option-as-alt).   

`macos-option-as-alt = true` - left and right `option` key is treated as `alt`
`macos-option-as-alt = left/right` - enable this for the left or right `option` key, respectively

**Comment by Y3K** [comment]
Exactly what I needed, thank you so much!
